mappings/net/minecraft/client/gui/screen/world/EditGameRulesScreen.mapping

```diff
@@ -50,3 +50,13 @@ CLASS net/minecraft/class_5235 net/minecraft/client/gui/screen/world/EditGameRul
 			METHOD method_27640 createRuleWidget (Lnet/minecraft/class_1928$class_4313;Lnet/minecraft/class_5235$class_5238;)V
 				ARG 1 key
 				ARG 2 widgetFactory
+	CLASS class_5400 NamedRuleWidget
+		FIELD field_25629 name Ljava/util/List;
```

Comment:
names?

Reply:
If this is a single name, why is this a list?

Reply:
@Shnupbups why is "name" a list?

Reply:
This is a `List<StringRenderable>` that acts as a name for this gamerule widget. I think it might also contain the description of the gamerule?

Reply:
I don't know why name is a list, phase might be right in that it also contains the description.

Reply:
Maybe they do the line wrapping logic before building the widget? I guess.